operator/internal/manifests/options.go

```diff
@@ -195,3 +198,15 @@ func calculateHTTPTimeouts(queryTimeout time.Duration) TimeoutConfig {
 		},
 	}
 }
+
+func ValidatePerTenantConfig(s *lokiv1.LimitsSpec) error {
+	if s != nil && s.Tenants != nil {
+		for _, config := range s.Tenants {
+			c := lokiv1.PerTenantLimitsTemplateSpec{}
+			if config == c {
+				return ErrInvalidPerTenantConfig
```

Comment:
So it's obvious to the user which key is causing the issue
```suggestion
				return fmt.Errorf("%w: %s", ErrPerTenantConfigInvalid, key)
```